x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/no_models.tsx

```diff
@@ -37,21 +41,28 @@ export const NoModelsPanel: React.FC = () => {
                 'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.noModels.description',
                 {
                   defaultMessage:
-                    'You have no trained models available. Please follow the documenation to add trained ml models to your cluster.',
+                    'You have no supported trained models available. Please follow the documentation to add trained machine learning models to your cluster.',
```

Comment:
```suggestion
                    'You have no trained models available. Please follow the documentation to add trained machine learning models to your cluster.',
```

Reply:
Should we be more explicit about the new link, i.e. something like:

"Learn more by reading [Getting started with Machine Learning](url) in the Enterprise Search documentation" ?

@serenachou

Reply:
@leemthompo The idea behind adding `supported` here is to give a clue that you could have Trained Models listed in ML but none that can be used by an inference pipeline. There was some confusion in some testing why someone saw this `No Models` screen but they did have 1 or more models in the Trained Models table in Machine Learning.

Reply:
@TattdCodeMonkey gotcha. I like your explanation here better!

"You have no trained models that can be used by an inference pipeline."

WDYT?

Reply:
I like that, thanks!

Reply:
@leemthompo what do you think about this:

![image](https://user-images.githubusercontent.com/1972968/203144037-61801e6b-fd50-41ae-b4bf-2130c965f65f.png)

I think I like having the link inline better than expanding the footer "Learn more" and the second paragraph does seem a unnecessary.

Reply:
We typically use a "Learn more" link to link to the  documentation.  Something like this:

>You have no machine learning models that can be use by an inference pipeline. [Learn how to add a trained model](url) (icon for outside docs)